astropy_helpers/openmp_helpers.py

```diff
@@ -73,7 +73,7 @@ def add_openmp_flags_if_available(extension):
 
         # Compile, link, and run test program
         ccompiler.compile(['test_openmp.c'], output_dir='objects', extra_postargs=[compile_flag])
-        ccompiler.link_executable(glob.glob(os.path.join('objects', '*')), 'test_openmp', extra_postargs=[link_flag])
+        ccompiler.link_executable(glob.glob(os.path.join('objects', '*' + ccompiler.obj_extension)), 'test_openmp', extra_postargs=[link_flag])
```

Comment:
Should this be ``'*.' + ccompiler.obj_extension`` or is the dot already included? (thinking of files that might end in ``c`` but not ``.c``)

Reply:
Ignore me, it includes the ``.``